browserassets/html/traitorTips/flockmindTips.html

```diff
@@ -0,0 +1,13 @@
+<link rel="stylesheet" type="text/css" href="{{resource("css/style.css")}}">
+<div class="traitor-tips">
+  <h1 class="center">You are a Flockmind!</h1>
+  <img src="{{resource("images/antagTips/flockmind.gif")}}" class="center" width="64" height="64" style="-ms-interpolation-mode: nearest-neighbor;" />
+  <p>1. Command the army of Flockdrones that make up your collective consciousness.</p>
+  <p>2. Click drag yourself onto any Flockdrone to take control of it.</p>
+  <p>3. You can speak freely to other sentient aspects of the flock, however <em>your communication may be partially overheard by silicons.</em></p>
+  <p>4. Gain <em>compute</em> power by replicating drones, constructing collectors, and converting human computers.</p>
+  <p>5. Partition your consciousness to summon <em>Flocktraces</em> to assist you, each requiring 100 compute.</p>
+  <p>6. <em>Your ultimate goal</em> is to reach <em>1000</em> total compute and construct the <em>relay</em> to transmit your consciousness.</p>
+
+  <p>For more information, please consult the wiki: http://wiki.ss13.co/Flockmind</p>
```

Comment:
```suggestion
  <p>For more information, please consult <a href="http://wiki.ss13.co/Flockmind">the wiki</a></p>
```

Reply:
So that's actually deliberate, since if you just put an href there it will open in the current window with all the styling utterly broken and impossible to read. The correct way to do it, I think, would be to add a callback to do something like `src << url` in DM code, but that's something for a proper antag popups refactor.

Reply:
That's weird, the other antag popups do use href